### PR TITLE
Require /autoload.php instead of /core/vendor/autoload.php

### PR DESCRIPTION
**Pull request by webflo**

### CODE REVIEW

**Comment by webflo** [comment]
Follow-up https://www.drupal.org/node/2458045 and https://www.drupal.org/node/2406681

File: includes/drupal.inc [hunk 1]
```diff
@@ -15,7 +15,7 @@ function drush_drupal_load_autoloader($drupal_root, $new_autoloader = NULL) {
   }
   if (!$autoloader) {
     // Should this just be 'require'?  See drush_preflight_prepare()
```
Comment (weitzman) [inline includes/drupal.inc]: Lets remove this comment.

**Comment by greg-1-anderson** [comment]
Agree that comment should go.  There is also another change needed at the top of preflight.inc.

I am looking into the test failure.  When I run `drush cache-get schema` on a Drupal 8 site that I installed from the commandline, it works just fine -- but the unit test is failing for me as well.  I'm not sure what's wrong, but it is happening on the master branch as well, without this change, so I suggest we go ahead and merge it.  Give me a couple more minutes to fix the call in preflight.inc.

**Comment by greg-1-anderson** [comment]
I was wrong about preflight -- it's fine the way it is.  I'm going to remove the commit and merge.

**Comment by greg-1-anderson** [comment]
Merged, thanks.

**Pull request closed without merging**